extensions/schema-compare/src/test/testContext.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
```

Comment:
should this file be names testcontext.test.ts - can you check how does a file not names *.test but inside test folder impact Coverage?

Reply:
@kisantia do you have any idea here around file names playing any role in CC?

Reply:
Anything that is part of test folder, isn't tested for Coverage, so the naming here doesn't matter. 
Moreover, *.test.ts files are the ones that actually have some tests that are run. So this wouldn't qualify.

Reply:
sounds good. thanks!